src/Sulu/Bundle/AdminBundle/Resources/js/components/Dialog/dialog.scss

```diff
@@ -0,0 +1,76 @@
+@import '../../containers/Application/colors.scss';
+
+$contentBackgroundColor: $concrete;
+$footerBackgroundColor: $white;
+$borderRadius: 3px;
+
+$titleColor: $black;
+$contentColor: $dustyGray;
+
+$confirmBackground: $shakespeare;
+$confirmColor: $white;
+
+$transitionDuration: 300ms;
+
+.container {
```

Comment:
Sry, now that we've been talking about that, I've seen that we have the same situation here 🙈 Would go for `dialog-container`.